fiscal_epos_print/static/src/js/chrome.js

```diff
@@ -0,0 +1,212 @@
+                self.chrome.loading_message(_t('Connecting to the fiscal printer'));
+                var printer_options = self.getPrinterOptions();
+                var fp90 = new eposDriver(printer_options, self);
+                fp90.getStatusOfFilesForADE();
+            });
+
+            this.$('.button.fiscal_closing').click(function() {
+                self.hide();
+                self.chrome.loading_show();
+                self.chrome.loading_message(_t('Connecting to the fiscal printer'));
+                var printer_options = self.getPrinterOptions();
+                var fp90 = new eposDriver(printer_options, self);
+                self.gui.show_popup('confirm', {
+                    'title': _t('Confirm Printer Fiscal Closure (Report Z)?'),
+                    'body': _t('Please confirm to execute the Printer Fiscal Closure'),
+                    confirm: function() {
+                        fp90.printFiscalReport();
+                    },
+                    cancel: function() {
+                        self.chrome.loading_hide();
+                    },
+                });
+            });
+
+            this.$('.button.fiscal_xreport').click(function() {
+                self.hide();
+                self.chrome.loading_show();
+                self.chrome.loading_message(_t('Connecting to the fiscal printer'));
+                var printer_options = self.getPrinterOptions();
+                var fp90 = new eposDriver(printer_options, self);
+                self.gui.show_popup('confirm', {
+                    'title': _t('Confirm Printer Daily Financial Report (Report X)?'),
+                    'body': _t('Please confirm to execute the Printer Daily Financial Report'),
+                    confirm: function() {
+                        fp90.printFiscalXReport();
+                    },
+                    cancel: function() {
+                        self.chrome.loading_hide();
+                    },
+                });
+            });
+
+            this.$('.button.open_cashdrawer').click(function() {
+                self.hide();
+                self.chrome.loading_show();
+                self.chrome.loading_message(_t('Connecting to the fiscal printer'));
+                var printer_options = self.getPrinterOptions();
+                var fp90 = new eposDriver(printer_options, self);
+                fp90.printOpenCashDrawer();
+            });
+
+            this.$('.button.reprint_last_receipt').click(function() {
+                self.hide();
+                self.chrome.loading_show();
+                self.chrome.loading_message(_t('Connecting to the fiscal printer'));
+                var printer_options = self.getPrinterOptions();
+                var fp90 = new eposDriver(printer_options, self);
+                self.gui.show_popup('confirm',{
+                    'title': _t('Confirm Reprint Last Receipt?'),
+                    'body': _t('Please confirm to print the last receipt'),
```

Comment:
Per non appesantire troppo (con il doppio confirm) suggerisco un semplice
Reprint Last Receipt ?
Please confirm to reprint the last receipt